frontend/src/lib/components/HedgehogBuddy/HedgehogBuddy.tsx

```diff
@@ -43,25 +42,53 @@ export type HedgehogBuddyProps = {
     hedgehogConfig?: HedgehogConfig
 }
 
+type Box = {
+    // Simplified rect based on bottom left xy and width/height
+    x: number
+    y: number
+    width: number
+    height: number
+}
+
+const elementToBox = (element: Element): Box => {
+    if (element === document.body) {
+        return {
+            x: 0,
+            y: -1000,
+            width: window.innerWidth,
+            height: 1000,
+        }
```

Comment:
why these values?

Reply:
Typical "gaming" logic. You build a floor by just having a big block at the bottom. Makes it easier to treat it like any other interactable. I'll add a comment to make that clearer